src/vs/workbench/contrib/remote/browser/tunnelView.ts

```diff
@@ -733,7 +733,17 @@ namespace ForwardPortAction {
 						}
 						remoteExplorerService.setEditable(undefined, null);
 					},
-					validationMessage: validateInput,
+					validationMessage: (value) => {
```

Comment:
Thanks for making the changes to tunnelView.ts! The code in both `validationMessage` here is duplicated. Can you refactor into a function in this file:
```typescript
function validationMessage(validationString: string | null): { content: string, severity: Severity } | null {
	if (!validationString) {
		return null;
	}

	return {
		content: validationString,
		severity: Severity.Error
	};
}
```
Then in the two `validationMessage` you just need to have:
```typescript
					validationMessage: (value) => {
						return validationMessage(validateInput(value));
					},
```